rllib/agents/sac/sac_tf_model.py

```diff
@@ -118,16 +122,16 @@ def build_action_model(self):
                 activation=getattr(tf.nn, self.actor_hidden_activation[i], None),
                 name="action_{}".format(i + 1))
             for i, hidden in enumerate(self.actor_hiddens)
-        ] + [
+         ] + [
             tf.keras.layers.Dense(
                 units=self.action_outs, activation=None, name="action_out")
-        ])
+         ])
         return action_model
 
     def build_q_net(self, name, observations, actions):
         # For continuous actions: Feed obs and actions (concatenated)
         # through the NN. For discrete actions, only obs.
-        if self.dropout_rate>0:
+        if self.dropout_rate > 0 or self.vib_regularizer["enable_regularizer"]:
```

Comment:
This if will break all existing tests that have a config without that key. I will check if the config is defined first.

Reply:
Thanks. Fixed it